docs/prometheus-grafana/prometheus-installation-configuration.md

```diff
@@ -0,0 +1,115 @@
+# Monitoring using Prometheus
+
+## Installing and configuring Prometheus for monitoring
+
+`k8ssandra-operator` has integrations with Prometheus which allow for the simple rollout of Prometheus ServiceMonitors for both Stargate and Cassandra Datacenters.
+
+### Prerequisites
+
+To use Prometheus for monitoring, you need to have the Prometheus operator installed on your Kubernetes (k8s) cluster. A simple way to install Prometheus operator is via [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus). 
+
+```
+kubectl -k https://github.com/prometheus-operator/kube-prometheus
```

Comment:
Sorry @jdonenine I only just saw this. I don't quite follow the question. Near as I can tell, this kustomize based install does the same thing as the helm chart, but without the helm dependancy. 

What benefits do you foresee by using Helm in this scenario? In particular, I have never used the Prometheus helm charts, so I'm not sure what you mean by this:

> That's a super broad deployment that will scrape all labels, so the example could be tuned to include only our monitors if we wanted, but it's pretty quick and straight forward.

Reply:
If someone is installing via Helm then I think kube-prometheus-stack makes sense. Maybe we should make it an optional dependency for the k8ssandra-operator chart.

Reply:
> If someone is installing via Helm then I think kube-prometheus-stack makes sense. Maybe we should make it an optional dependency for the k8ssandra-operator chart.

That's probably true. I thought we wanted avoid assuming a helm dependancy though? This is where it might be good to [have both](https://k8ssandra.atlassian.net/browse/K8SSAND-1281) as I suggested here.

Reply:
How would we avoid a Helm dependency? Considering k8ssandra 1.x is completely Helm-based, I would venture to guess that a lot of users will install with Helm.

I agree with your suggestion to have examples for both.

Reply:
>What benefits do you foresee by using Helm in this scenario? I have never used the Prometheus helm charts, so I'm not sure what you mean by this in particular:

The benefit is just that generally it's desired, with so much of the original K8ssandra provided via helm I'd even so much as call it an expectation.

In this case though, I don't have any reason to say you shouldn't go ahead with what you have here... the helm path provides an alternate means, in this particular scenario a particularly easy one for users.